programs/survival/vacuum.sc

```diff
@@ -0,0 +1,52 @@
+//first shown on Xisumavoid's YouTube channel
+//https://youtu.be/FMu8T8KriQY
+__config() ->
+(
+	return(m(l('scope','player'),l('stay_loaded',true)))
+);
+
+__move_items() ->
+(
+	loop(41,
+		//check to see if the item is a single shulker box named 'Vacuum'
+		if(inventory_get(player(), _):0 ~ 'shulker_box' && parse_nbt(parse_nbt(inventory_get(player(),_):2):'display':'Name'):'text' == 'Vacuum' && inventory_get(player(),_):1 == 1,
```

Comment:
I would suggest avoiding excessive parsing of entire shulkerbox nbt just to get its name, and also fetching the nbt only once. That should significantly cut down on compute